### PR TITLE
Update dashboard_contributors.xml

### PR DESCRIPTION
**Pull request by Technologx**
Changed my link from Github to G+

### CODE REVIEW

File: core/src/main/res/xml/dashboard_contributors.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <contributor
         name="Technologx"
         contribution=""
-        image="https://avatars0.githubusercontent.com/u/17693282?v=3&amp;s=400"
-        link="https://github.com/Technologx" />
+        image="https://lh3.googleusercontent.com/xzdaNykgAewl8vNKG-CIxeBGWUnr02NV-KBTHRKBCiGD_5ydpBzBGRMep5Nu5MO3G7wcvCFjYLUzEw=w1280-h800-no"
```
Comment (danimahardhika) [inline core/src/main/res/xml/dashboard_contributors.xml]: Reduce the resolution, something like https://lh3.googleusercontent.com/xzdaNykgAewl8vNKG-CIxeBGWUnr02NV-KBTHRKBCiGD_5ydpBzBGRMep5Nu5MO3G7wcvCFjYLUzEw=w300-no

Comment (Technologx) [inline core/src/main/res/xml/dashboard_contributors.xml]: Feel free to use that one I don't mind

Comment (danimahardhika) [inline core/src/main/res/xml/dashboard_contributors.xml]: You need to change it, create another commit

**Comment by Technologx** [comment]
Alright I changed it for you

**Pull request merged**